hazelcast/src/main/java/com/hazelcast/internal/serialization/impl/ByteArrayObjectDataOutput.java

```diff
@@ -429,11 +429,8 @@ public ByteOrder getByteOrder() {
 
     @Override
     public String toString() {
-        final StringBuilder sb = new StringBuilder();
-        sb.append("ByteArrayObjectDataOutput");
-        sb.append("{size=").append(buffer != null ? buffer.length : 0);
-        sb.append(", pos=").append(pos);
-        sb.append('}');
-        return sb.toString();
+        return "ByteArrayObjectDataOutput"
+                + "{size=" + (buffer != null ? buffer.length : 0)
+                + ", pos=" + pos + '}';
```

Comment:
The opening `{` should be in the first line in `"ByteArrayObjectDataOutput"`.

Normally we have the ` + '}';` on a new line on multi-line messages.